src/metrics/metadata/metadata.go

```diff
@@ -280,21 +282,26 @@ func (metadatas PipelineMetadatas) ApplyOrRemoveDropPolicies() (
 	return result, RemovedIneffectiveDropPoliciesResult
 }
 
-// ApplyCustomTags applies custom M3 tags.
-func (metadatas PipelineMetadatas) ApplyCustomTags() (
-	dropTimestamp bool,
-) {
-	// Go over metadatas and process M3 custom tags.
+// ShouldDropTimestamp applies custom M3 tags.
+func (metadatas PipelineMetadatas) ShouldDropTimestamp(untimedRollups bool) bool {
```

Comment:
nit: Since this is a single boolean I'd opt to make this a struct so it's clear from the "caller" perspective of what the boolean means.

i.e.

```
type ShouldDropTimestampOptions struct {
  UntimedRollups bool
}
```